models/servicenow__ticket_enhanced.sql

```diff
@@ -65,15 +65,18 @@ select
   task.task_state,
   task.task_number,
   task.task_order,
-  case when problem_task.problem_task_id is not null then true else false end as is_problem_task,
+  case when problem_task.problem_task_id is not null 
+    then true 
+    else false 
+  end as is_problem_task,
   case when change_task.change_task_id is not null then true else false end as is_change_task,
   task.task_created_at,
   task.sys_created_by,
-  creator.email as creator_email,
-  creator.manager_value as creator_manager_value,
-  creator.department_value as creator_department_value,
-  creator.sys_user_name as creator_name,
-  creator.roles as creator_roles,
+  creator.email as creator_email, 
+  creator.manager_value as creator_manager_value, 
+  creator.department_value as creator_department_value, 
+  creator.sys_user_name as creator_name, 
+  creator.roles as creator_roles, 
```

Comment:
Since all these people-related fields come from the same sys_user table, the only way I could think to be able to apply all the field renaming in staging would be to expand the task staging model in order to join on itself multiple times, which would be pretty extensive